examples/createContract/src/spec.md

```diff
@@ -6,13 +6,18 @@ interface create_kid0()
 
 types
 
-    KID : address Kid
-    Id  : uint256
+    KID          : address Kid
+    Id           : uint256
+    Initialized  : uint256
 
 creates storage KID
 
-    #Kid.parent  |-> ACCT_ID
-    #Kid.ID      |-> Id
+    #Kid.parent      |-> ACCT_ID
+    #Kid.ID          |-> Id
+
+storage KID
+
+    #Kid.initialized |-> _ => Initialized
```

Comment:
@mhhf I realise this isn't currently supported - do you think this is the best approach for dealing with modifications to new contrac storage? Or could it be handled by the `creates storage` block?

Reply:
I'd just state `#Kid.initialized |-> Initialized`
dirtyness of act semantic make it look bad but that what it bouls down to since we only care abot the state "at the end of the execution"